piDIOComm.c

```diff
@@ -151,70 +154,69 @@ INT32U piDIOComm_Init(INT8U i8uDevice_p)
 
 INT32U piDIOComm_sendCyclicTelegram(INT8U i8uDevice_p)
 {
-	INT32U i32uRv_l = 0;
-	SIOGeneric sRequest_l;
-	SIOGeneric sResponse_l;
-	INT8U len_l, data_out[18], i, p, data_in[70];
+	static INT8U last_out[40][DIO_OUTPUT_DATA_LEN];
+
+	INT8U i, p, data_in[70];
 	INT8U i8uAddress;
```

Comment:
Hm, can we change the data type to the kernel-native `u8` in those lines that are touched anyway, so that we get rid of these alien-looking data types over time?

BTW is there a reason for the newly inserted blank line in the middle of variable declarations?

Reply:
Its a separation from the static to the local variable definitions